Makefile

```diff
@@ -108,10 +108,9 @@ bundle: operator-sdk manifests
 build-bundle:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
-deploy-olm:
-	olm_latest_version=$$(curl -s $(OLM_URL) | grep tag_name | grep -v -- '-rc' | head -1 | awk -F': ' '{print $$2}' | sed 's/,//' | xargs) ;\
-	kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/$${olm_latest_version}/crds.yaml ;\
-	kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/$${olm_latest_version}/olm.yaml ;\
+deploy-olm: operator-sdk
+	operator-sdk olm install
```

Comment:
i guess we can delete the OLM_URL variable

Reply:
done